planner/core/logical_plan_builder.go

```diff
@@ -3329,6 +3329,9 @@ func (b *PlanBuilder) buildWindowFunctions(ctx context.Context, p LogicalPlan, g
 // Because of the grouped specification is different from it, we should especially check them before build window frame.
 func (b *PlanBuilder) checkOriginWindowSpecs(funcs []*ast.WindowFuncExpr, orderByItems []property.Item) error {
 	for _, f := range funcs {
+		if f.IgnoreNull {
```

Comment:
The `Distinct` and `FromLast` should also be checked, you can do it in this pr or another pr.

Reply:
OK, I'll do it in this pr.